internal/sdkv2provider/resource_cloudflare_waiting_room.go

```diff
@@ -64,6 +64,7 @@ func buildWaitingRoom(d *schema.ResourceData) cloudflare.WaitingRoom {
 		CookieSuffix:            d.Get("cookie_suffix").(string),
 		AdditionalRoutes:        additional_routes,
 		QueueingStatusCode:      d.Get("queueing_status_code").(int),
+		EnabledOriginCommands:   d.Get("enabled_origin_commands").([]string),
```

Comment:
cty unfortunately doesn't have the concept of `[]string` so instead, you'll need to wrap this in `expandInterfaceToStringList` and cast to a generic `[]interface{}`.
```suggestion
		EnabledOriginCommands:   expandInterfaceToStringList(d.Get("enabled_origin_commands").([]interface{})),
```